src/components/WorldListPage_WorldsForEnsOwnersFeature/WorldsStorage/WorldsStorage.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import classNames from 'classnames'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Props } from './WorldsStorage.types'
+import styles from './WorldsStorage.module.css'
+
+export const CURRENT_MBS_TEST_ID = 'worlds-storage-current-mbs'
+export const BAR_FRONT_TEST_ID = 'worlds-storage-bar-front'
+
+const WorldsStorage = ({ maxBytes, currentBytes }: Props) => {
+  const maxMbs = maxBytes / 1000000
+  const currentMbs = currentBytes / 1000000
+  const usedPercentage = (currentMbs * 100) / maxMbs
+
+  return (
+    <div className={styles.worldsStorage}>
+      <div className={styles.spaceContainer}>
+        <span>{t('worlds_list_page.worlds_storage.space_used')}</span>
+        <div data-testid={CURRENT_MBS_TEST_ID}>
+          <span className={styles.currentMbs}>{currentMbs.toFixed(2)}</span> / {maxMbs.toFixed(2)} mb
+        </div>
+      </div>
+      <div className={styles.barContainer}>
+        <div className={classNames(styles.bar, styles.barBehind)} />
+        <div
+          data-testid={BAR_FRONT_TEST_ID}
+          style={{
+            width: `${Math.trunc(usedPercentage)}%`
+          }}
+          className={classNames(styles.bar, styles.barFront)}
+        />
```

Comment:
[Semantic-ui has a progress bar](https://react.semantic-ui.com/modules/progress/) did you have the chance to check it out? We're using it when importing multiple wearables in the third party components.

Reply:
Replaced by semantic progress bar.

Ty :D